packages/protocol/analysisManager.ts

```diff
@@ -208,11 +209,11 @@ class AnalysisManager {
   };
 
   private readonly onAnalysisPoints = ({ analysisId, points }: analysisPoints) => {
-    const handler = this.handlers.get(analysisId);
-    if (handler != null && typeof handler.onAnalysisPoints === "function") {
-      points.forEach(point => ThreadFront.updateMappedLocation(point.frame));
-      handler.onAnalysisPoints(points);
-    }
+    points.forEach(point => {
+      ThreadFront.updateMappedLocation(point.frame);
```

Comment:
I changed this line. I was a little confused - don't we *always* want to update the frame locations, even if we don't happen to have a handler for this analysis?

Reply:
FYI Brian, Holger, and I had a discussion about this last week. We're probably going to _stop_ mutating frame locations shortly - there's only a few places where those locations get used.

Reply:
Very good, I'll leave it then! Thanks!